--- qtpfsgui-1.8.12/src/Libpfs/pfs.cpp.orig	2008-07-21 13:10:44.097801066 +0200
+++ qtpfsgui-1.8.12/src/Libpfs/pfs.cpp	2008-07-21 13:11:25.594949061 +0200
@@ -27,6 +27,7 @@
  * $Id$
  */
 #include <map>
+#include <cstdlib>
 
 #include "pfs.h"
 
--- qtpfsgui-1.8.12/src/TM_operators/ashikhmin02/tmo_ashikhmin02.cpp.orig	2008-07-21 13:14:27.582441477 +0200
+++ qtpfsgui-1.8.12/src/TM_operators/ashikhmin02/tmo_ashikhmin02.cpp	2008-07-21 13:14:40.081582440 +0200
@@ -29,6 +29,7 @@
 #include <iostream>
 #include <math.h>
 #include <assert.h>
+#include <cstdlib>
 
 #include "../../Libpfs/array2d.h"
 #include "tmo_ashikhmin02.h"
