--- qtpfsgui-1.9.3/qtpfsgui.desktop~	2008-10-15 19:44:49.000000000 +0200
+++ qtpfsgui-1.9.3/qtpfsgui.desktop	2010-05-11 16:44:21.908102393 +0200
@@ -7,7 +7,7 @@
 GenericName[en_US]=HDR imaging
 GenericName[ru]=Работа с HDR-фотографией
 GenericName[it]=Workflow per immagini HDR
-Categories=Graphics;RasterGraphics;Photography;Viewer;Qt;
+Categories=Graphics;RasterGraphics;Photography;Qt;
 Comment=Qtpfsgui is a graphical user interface used to create and tonemap HDR images
 Comment[de]=Qtpfsgui ist eine grafische Benutzeroberfläche um HDR Bilder zu erzeugen und um die Farbabbildung durchzuführen
 Comment[en_US]=Qtpfsgui is a graphical user interface used to create and tonemap HDR images
